homework/nwd-nnw/nwdNww.hpp

```diff
@@ -1,11 +1,42 @@
 #pragma once
 
 int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    if(lhs < 0){
+        lhs = -lhs;
+    }
+    if(rhs < 0){
+        rhs = -rhs;
+    }
+    if(lhs == rhs){
+        return lhs;
+    }
+    int max = rhs;
+    if(lhs > rhs){
+        max = lhs;
+    }
```

Comment:
Use std::abs 
and
Use std::swap

if (lhs > rhs) {
   std::swap(lhs, rhs)
}